alma/alma-installments-prestashop
# Handle ps account & PS 1.6

### Reason for change

[Linear task](https://linear.app/almapay/issue/ECOM-1673/[%F0%9F%A7%A9-ps]-handle-the-display-of-the-ps-link-only-if-the-module-ps)

### Code changes
![image](https://github.com/alma/alma-installments-prestashop/assets/110386752/d928697a-2123-4370-bfe7-e1bb1b5014d7)

### How to test
On PS 1.6 you must have the suggest PS account module message, then when you install the module, you see the module
